pkg/configmap/configmap.go

```diff
@@ -173,7 +173,7 @@ func defaultKueueConfigurationTemplate(kueueCfg kueue.KueueConfiguration) *confi
 		FeatureGates: map[string]bool{
 			// Disable the HierarchicalCohorts feature gate by default.
 			// related to https://github.com/kubernetes-sigs/kueue/issues/4869
-			"HierarchialCohorts": false,
+			"HierarchicalCohorts": false,
```

Comment:
Can you check this?

I had to rename as later versions of kueue 0.11 renamed this.